_posts/2020-03-26-CollegeBreaks.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "HOW TO UTILISE COLLEGE BREAKS EFFECTIVELY"
+author: Arka
+categories: [ College Breaks, Getting Started, Time Utilization]
+image: assets/images/CollegeBreak.jpg
+last_added: true
+hidden: false
+---
+![](https://github.com/monsij/insight/raw/master/assets/images/mainpic.jpg)
```

Comment:
Add the paragraph "After our semester examinations..." from line 10. Also do not use any header for Introductory parapgraph i.e "##Introduction"The main article picture rendered is automatically taken from the thumbnail ( which is `assets/images/CollegeBreak.jpg` in this case). You can place `assets/images/mainpic.jpg` after the introductory text.